### ISSUE TITLE
Stack traces too short

### ISSUE THREAD

**Issue by ohAitch** [urbit/urbit#672]
The solution in #600 was too eager, resulting in the relevant frames being snipped out:

```
!:
|^  ^-  *
    foo
++  foo  !!
--
/~fyr/home/~2016.1.13..23.40.06..9c63:<[2 5].[3 8]>
/~fyr/home/~2016.1.13..23.40.06..9c63:<[4 10].[4 12]>
```

vs full

```
/~zod/home/~2016.1.13..23.39.42..831f:<[2 1].[5 3]>
/~zod/home/~2016.1.13..23.39.42..831f:<[2 5].[3 8]>
/~zod/home/~2016.1.13..23.39.42..831f:<[3 5].[3 8]>
/~zod/home/~2016.1.13..23.39.42..831f:<[4 10].[4 12]>
```

We don't actually care that much about the `<[2 5].[3 8]>`; the important frames are `[3 5..8]` and `[4 10..12]`. The shortening code needs to be debugged to account for this.

**Comment by galenwp** [comment]
@ohAitch didn't we revert this?

**Comment by ohAitch** [comment]
Yeah, I think this should be closed and #600 reopened or sth

**Comment by galenwp** [comment]
Please do?
